src/libslic3r/GCode/GCodeProcessor.cpp

```diff
@@ -476,6 +478,12 @@ void GCodeProcessor::TimeProcessor::post_process(const std::string& filename, st
         if (line.length() > 1) {
             line = line.substr(1);
             if (line == reserved_tag(ETags::First_Line_M73_Placeholder) || line == reserved_tag(ETags::Last_Line_M73_Placeholder)) {
+                if (disable_m73) {
```

Comment:
Why do we remove M73 lines here?
It should be sufficient to skip emitting M73 code in writer, like what PrusaSlicer did?

Reply:
Just in case someone copy paste machine gcode somewhere that contains M73 commands?